src/screens/SendSheet.js

```diff
@@ -813,6 +812,12 @@ export default function SendSheet(props) {
 
   const isEmptyWallet = !sortedAssets?.length && !sendableUniqueTokens?.length;
 
+  const filteredUserAccountsFromContacts = useMemo(() => {
+    return userAccounts.filter(
+      account => !filteredContacts.some(contact => contact.address.toLowerCase() === account.address.toLowerCase())
+    );
+  }, [userAccounts, filteredContacts]);
+
```

Comment:
fixes showing the account in the user wallets list AND contacts list if it was there.